modules/test/playwright/tests/journal-web/journalArticle.spec.ts

```diff
@@ -1011,6 +1011,68 @@ translationTest(
 	}
 );
 
+translationTest(
+	'Translate the Rich Text field and check if the translation persists after coming back to the page',
+	async ({journalEditArticlePage, journalPage, page, site}) => {
+		await journalPage.goto();
+
+		await journalEditArticlePage.goto({siteUrl: site.friendlyUrlPath});
+
+		const title = getRandomString();
+
+		await journalEditArticlePage.fillTitle(title);
+
+		const englishContent = 'English Language Text';
+
+		const catalanContent = 'Catalan Language Text';
+
+		await journalEditArticlePage.fillContent(englishContent);
+
+		const translationButton = page.getByRole('combobox', {
+			name: 'Select a language',
+		});
+
+		await clickAndExpectToBeVisible({
+			autoClick: true,
+			target: page.getByRole('option', {
+				name: 'Catalan Language: Not Translated',
+			}),
+			trigger: translationButton,
+		});
+
+		await journalEditArticlePage.fillContent(catalanContent);
+
+		await expect(
+			journalEditArticlePage.journalPage.articleContentTextBox.getByText(
+				catalanContent
+			)
+		).toBeVisible();
+
+		await journalEditArticlePage.publishButton.click();
+
+		await waitForSuccessAlert(
+			page,
+			`Success:${title} was created successfully.`
+		);
+
+		await page.getByRole('link', {name: title}).click();
+
+		await clickAndExpectToBeVisible({
+			autoClick: true,
+			target: page.getByRole('option', {
+				name: 'Catalan Language: Not Translated',
```

Comment:
here the name should be `{ name: 'Catalan Language: Translating 1/2' }` as teh catalan language has the rich text field translated, this is the reason the test fails